$OpenBSD: patch-ghc_rts_Makefile,v 1.3 2003/09/01 20:32:14 espie Exp $
Tell the bootstrap code how to compile the .s file
This code replaces the silly [] of code used in Adjustor.c

--- ghc/rts/Makefile.orig	Thu Jul 24 22:17:59 2003
+++ ghc/rts/Makefile	Thu Aug 21 13:45:54 2003
@@ -53,6 +53,10 @@
 HC_SRCS = $(filter %.hc, $(SRCS))
 HC_OBJS = $(patsubst %.hc,%.$(way_)o, $(HC_SRCS))
 
+C_OBJS += obscure_ccall.o
+obscure_ccall.o :
+	$(CC) -c obscure_ccall.s
+
 CLEAN_FILES += $(HC_OBJS)
 
 # Override the default $(LIBOBJS) (defaults to $(HS_OBJS))
